lib/offline_sort/offline_sort.rb

```diff
@@ -4,8 +4,8 @@
 require 'tempfile'
 
 module OfflineSort
-  def self.sort(*args, &sort_by)
-    Sorter.new(*args, &sort_by).sort
+  def self.sort(enumerable, **kwargs, &sort_by)
```

Comment:
This is not exactly the same way Salsify fixed this issue, but this is actually better. Salsify used `*args, **kwargs`, but if no arguments are passed at all, that ends up passing through `{}` as the required first argument `enumerable` (and using the defaults for all the keyword arguments) which essentially makes the first argument optional and is clearly not the intent. The solution in this PR enforces that the first, positional argument is required.

Reply:
Good idea!